modules/enabling-OVS-balance-slb-mode.adoc

```diff
@@ -0,0 +1,180 @@
+* Supports the OVN-Kubernetes CNI plugin and easily integrates with the plugin.
+* Natively supports `balance-slb` mode. 
+
+.Prerequisites
+
+* You have more than one physical interface attached to your primary network and you defined the interfaces in a `MachineConfig` file.
+* You created a manifest object and defined a customized `br-ex` bridge in the object configuration file.
+* You have more than one physical interfaces attached to your primary network and you defined the interfaces in a NAD file.
+
+.Procedure
+
+. For each bare-metal host that exists in a cluster, in the `install-config.yaml` file for your cluster define a `networkConfig` section similar to the following example:
++
+[source,yaml]
+----
+# ...
+networkConfig:
+  interfaces:
+    - name: enp1s0 <1>
+      type: interface
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        enabled: false
+    - name: enp2s0 <2>
+      type: interface
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        enabled: false
+    - name: enp3s0 <3>
+      type: interface
+      state: up
+      ipv4:
+        enabled: false
+      ipv6:
+        enabled: false
+# ...
+----
+<1> The interface for the provisioned network interface controller (NIC). 
+<2> The first bonded interface that pulls in the Ignition config file for the bond interface.
+<3> The second bonded interface is part of a minimal configuration that pulls ignition during cluster installation.
+
+. Define each network interface in an NMState configuration file:
++
+.Example NMState configuration file that defines many network interfaces
+[source,yaml]
+----
+ovn:
+  bridge-mappings:
+    - localnet: localnet-network
+      bridge: br-ex
+      state: present
+interfaces:
+  - name: br-ex
+    type: ovs-bridge
+    state: up
+    ipv4:
+      enabled: false
+      dhcp: false
+    ipv6:
+      enabled: false
+      dhcp: false
+    bridge:
+      allow-extra-patch-ports: true
+      port:
+        - name: br-ex
+        - name: patch-ex-to-phy
+    ovs-db:
+      external_ids:
+        bridge-uplink: "patch-ex-to-phy"
+  - name: br-ex
+    type: ovs-interface
+    state: up
+    copy-mac-from: enp2s0
+    ipv4:
+      enabled: true
+      dhcp: true
+      auto-route-metric: 48
+    ipv6:
+      enabled: false
+      dhcp: false
+  - name: br-phy
+    type: ovs-bridge
+    state: up
+    ipv4:
+      enabled: false
+      dhcp: false
+    ipv6:
+      enabled: false
+      dhcp: false
+    bridge:
+      allow-extra-patch-ports: true
+      port:
+        - name: patch-phy-to-ex
+        - name: ovs-bond
+          link-aggregation:
+            mode: balance-slb
+            port:
+              - name: enp2s0
+              - name: enp3s0
+  - name: patch-ex-to-phy
+    type: ovs-interface
+    state: up
+    patch:
+      peer: patch-phy-to-ex
+  - name: patch-phy-to-ex
+    type: ovs-interface
+    state: up
+    patch:
+      peer: patch-ex-to-phy
```

Comment:
We need to add back in the bond ports to set the MTU.

```yaml
    - name: enp2s0
      type: ethernet
      state: up
      mtu: 1500  <1> 
      ipv4:
        enabled: false
      ipv6:
        enabled: false
    - name: enp3s0
      type: ethernet
      state: up
      mtu: 1500  <1> 
      ipv4:
        enabled: false
      ipv6:
        enabled: false

<1> `br-ex` MTU as set by DHCP must be manually set on the bond ports 


```

The DHCP MTU settings will not propagate down to the bond ports. If we set the MTU using DHCP we have to manually set the MTU on the bond ports to match.

Reply:
:heavy_check_mark:

Reply:
I don't see this yet.

Reply:
The difference between step 1 and this step is that in step 1 we have DHCP on `enp2s0` but once we create `br-ex` we disable `ipv4` `ipv6` on `enp2s0` 

In general it's better to always include the full config in a single NMstate, so it's okay that we duplicate the `enp1s0` and `enp3s0` configs here.  We like to configure the entire network at once.